DetectionFactory/ConvertDetection.cs

```diff
@@ -0,0 +1,34 @@
+using Business.Detection.Common.Models;
+using Data;
+using Data.Detection;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace FactoryDetectionTests
+{
+    public class ConvertDetection
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void DetectionFactoryTest()
```

Comment:
Should we rename this test to look like this: "public void Method_Scenario_Expected()". Ex: "public void Post_StatusCode_IsRight()"

Reply:
Done